### PR TITLE
Dump all stored documents as a JSON file

### PR DESCRIPTION
**Pull request by Phyks**
Hi,

Here is a beginning of fix required for https://github.com/konnectors/libs/issues/176. I implemented dumping the stored documents for now, and I'm waiting for some review on this to implement the possibility of passing fields as environment variables. I think smaller PRs will be easier to process :)

Thanks!

EDIT: For now it simply dumps imported documents in a `importedData.json` file in the current working dir.

### CODE REVIEW

**Comment by Phyks** [comment]
Sorry, I spotted a few bugs. Should be fixed now. It is quite complicated to test changes in this repo with Konnectors for me, my current setup involves `link`ing the `cozy-konnectors-lib` subpackage into the konnector repo and running manually the `cozy-standalone` binary from the subpackage here. Is there a better setup / tooling to test this lib? Thanks!

**Comment by Phyks** [comment]
> Clean the JSON store each time (I prefer this one, what do you think ?)

For my use case, it does not matter, I would have purged the file afterwards anyways. Definitely fine for me, I edited accordingly.


> And here is some vision of what I would like to do for cozy-konnector-libs :grinning: 

This looks super promising! For my use case for now, doing some post-processing is perfectly fine, so I'll just parse the dump JSON I think :)


I should have made the edits you requested, let me know if something is missing.

Thanks!

**Comment by Phyks** [comment]
Hi,

Any updates on this? Thanks!

File: packages/cozy-konnector-libs/src/helpers/cozy-client-js-stub.js [hunk 1]
```diff
@@ -13,20 +13,65 @@ if (fs.existsSync(FIXTURE_PATH)) {
...
 }
 
+let DUMP_PATH = 'importedData.json'
+const KONNECTOR_DEV_CONFIG_PATH = path.resolve('konnector-dev-config.json')
+if (fs.existsSync(KONNECTOR_DEV_CONFIG_PATH)) {
+  const KONNECTOR_DEV_CONFIG = require(KONNECTOR_DEV_CONFIG_PATH)
+  DUMP_PATH = path.join(KONNECTOR_DEV_CONFIG.fields.folderPath || '.', DUMP_PATH)
+}
+// Truncate dump file
+fs.writeFileSync(DUMP_PATH, '', 'utf8')
```
Comment (doubleface) [inline packages/cozy-konnector-libs/src/helpers/cozy-client-js-stub.js]: @Phyks Sorry for the long delay, please just replace : 

```javascript
fs.writeFileSync(DUMP_PATH, '', 'utf8')
```

with : 

```javascript
fs.writeFileSync(DUMP_PATH, '[]', 'utf8')
```

Or else you get a `JSON.parse` error.

**Comment by Phyks** [comment]
Thanks for the review! I did the required changes.

**Comment by doubleface** [comment]
Thanks for this long awaited PR @Phyks !

**Comment by Phyks** [comment]
Thanks! Do you know when the next version of this lib will be released?

**Comment by doubleface** [comment]
This won't long

**Pull request merged**